pkg/provision/storage/shared.go

```diff
@@ -61,7 +61,7 @@ func WorkspaceNeedsStorage(workspace *dw.DevWorkspaceTemplateSpec) bool {
 }
 
 func getPVCSpec(name, namespace string, storageClass *string, size resource.Quantity) (*corev1.PersistentVolumeClaim, error) {
-
+	
```

Comment:
If possible, please remove this tab (unless it's coming from go format)

Reply:
oops, fixed it in the amended commit